docs/app/Examples/elements/List/Content/ListImageExample.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListImageExample extends Component {
+  render() {
+    const helenAvatar = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/1' />;
```

Comment:
There is an `<Image />` component, let's use only Stardust components if we can.